index.js

```diff
@@ -79,5 +31,85 @@ const createCucumber = (spec, definitions) => {
       }
     )
   }`
-  return cucumberFile
-}
\ No newline at end of file
+);
+
+const pattern = `${process.cwd()}/cypress/support/step_definitions/**.js`;
+const stepDefinitionsPaths = [].concat(glob.sync(pattern));
+
+const compile = (spec) => {
+  log('compiling', spec);
+
+  const definitions = [];
+  stepDefinitionsPaths.forEach((path) => {
+    definitions.push(
+      `{ ${fs.readFileSync(path).toString().replace('cypress-cucumber-preprocessor', 'cypress-cucumber-preprocessor/resolveStepDefinition')}
+      }`
+    );
+  });
+
+  return createCucumber(spec, JSON.stringify(definitions));
+};
+
+
+const touch = (filename) => {
+  fs.utimesSync(filename, new Date(), new Date());
+};
+
+
+const transform = (file) => {
+  let data = '';
+
+  function write(buf) { data += buf; }
+  function end() {
+    if (file.match('.feature$')) {
+      log('compiling feature ', file);
+      this.queue(compile(data));
+    } else {
+      this.queue(data);
+    }
+    this.queue(null);
+  }
+
+  return through(write, end);
+};
+
+
+const preprocessor = (options) => {
```

Comment:
do you have something against using (options = browserify.defaultOptions) here instead? I think this way we could keep the backward compatibility and make it as easy to set up as it already was (and if someone wants to customise - there is a way).
Otherwise - great work :) this makes this plugin really usable :) and we are still just at the beginning of the journey

Reply:
Good idea, I'll make the change